src/Fixer/Basic/BracesFixer.php

```diff
@@ -323,13 +323,19 @@ function ($item) {
                     continue;
                 }
 
-                if (1 === $nestLevel && $nestToken->equalsAny(array(';', '}'))) {
+                if (1 === $nestLevel && $nestToken->equalsAny(array(';', '}', array(T_COMMENT)))) {
                     $nextNonWhitespaceNestIndex = $tokens->getNextNonWhitespace($nestIndex);
                     $nextNonWhitespaceNestToken = $tokens[$nextNonWhitespaceNestIndex];
 
                     if (
                         // next Token is not a comment
-                        !$nextNonWhitespaceNestToken->isComment() &&
+                        (
+                            !$nextNonWhitespaceNestToken->isComment() ||
+                            (
+                                substr($nextNonWhitespaceNestToken->getContent(), 0, 2) === '//' &&
```

Comment:
I think this will not work for the single character comment case `#`, could you add a test for this?